### ISSUE TITLE
look through all skiped acceptance tests and find out if the issues are fixed

### ISSUE THREAD

**Issue by individual-it** [owncloud/core#33383]
We have a lot of tests that were skipped for various reasons, lets have a look if the reasons are still valid. Maybe the bugs are already fixed and the tests can be unskipped

List of issues that have skipped tests in core:
```
files=`find -name *.feature`; for i in `grep "@skip @issue" $files | cut -d ":" -f2 | cut -d "-" -f2 | cut -d" " -f1 | sort  | uniq`; do echo "#$i"; done
```
#23151
#29016
#29599
#31015
#31276
#32068
#32385
#32593
#32670
#32732
#32868
#33030
#33093

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks the contributor most likely able to help you is @phil-davis.

Possibly related issues are https://github.com/owncloud/core/issues/21767 (issue), https://github.com/owncloud/core/issues/22485 (test), https://github.com/owncloud/core/issues/22492 (test), https://github.com/owncloud/core/issues/31579 (test), and https://github.com/owncloud/core/pull/32378 (Small fixes in acceptance tests).

**Comment by individual-it** [comment]
fixed by #33393
